selfdrive/athena/athenad.py

```diff
@@ -274,7 +315,7 @@ def getMessage(service=None, timeout=1000):
   if ret is None:
     raise TimeoutError
 
-  return ret.to_dict()
+  return cast(Dict[str, Any], ret.to_dict())
```

Comment:
need to check whether or not cast was actually required

Reply:
I think it's required, would need to update `capnp.lib.capnp._DynamicStructReader` typing